util/connection/wrappers/async/async_errors.go

```diff
@@ -21,15 +21,17 @@
 package async
 
 import (
+	"errors"
 	"fmt"
 )
 
+var v ErrorAsyncJobInProgress
```

Comment:
Are there any specific reasons why this was added to package scope and not into IsAsyncJobInProgress func?
Also not sure if this is thread-safe in that way?

Reply:
fixed